src/pages/instances/InstanceList.tsx

```diff
@@ -124,22 +151,32 @@ const InstanceList: FC = () => {
   };
 
   const filteredInstances = instances.filter((item) => {
-    if (query) {
-      const q = query.toLowerCase();
+    if (queries.length > 0) {
+      const keywords = queries.map((q) => q.toLowerCase());
       if (
-        !item.name.toLowerCase().includes(q) &&
-        !item.status.toLowerCase().includes(q) &&
-        !item.type.toLowerCase().includes(q) &&
-        !item.description.toLowerCase().includes(q) &&
-        !item.config["image.description"]?.toLowerCase().includes(q)
+        !keywords.every((keyword) =>
+          item.name.toLowerCase().includes(keyword)
+        ) &&
+        !keywords.every((keyword) =>
+          item.description.toLowerCase().includes(keyword)
+        )
       ) {
         return false;
       }
     }
-    if (type !== "any" && item.type !== type) {
+    if (
+      statuses.length > 0 &&
+      !statuses.some((status) => item.status === status)
```

Comment:
`!statuses.includes(item.status)`

Reply:
This could have worked, but eventually I decided to implement the "merged" logic for temporary statuses.